tests/foreman/smoke/test_cli_smoke.py

```diff
@@ -729,5 +729,6 @@ def test_end_to_end(self):
                 .format(result.stderr, result.return_code)
             )
             # Verify if package is installed by query it
-            result = vm.run('rpm -q {0}'.format(package_name))
+            result = vm.run('rpm -qa | grep {0}'.format(package_name))
+            self.assertTrue(result.stdout)
```

Comment:
Better to leave the previous `rpm -q ...` command and assert for the return code:

```
$ rpm -q vim
package vim is not installed
$ echo $?
1
$ rpm -q less
less-458-8.el7.x86_64
$ echo $?
0
```